scipy/stats/tests/test_stats.py

```diff
@@ -538,6 +538,98 @@ def test_gh3014(self):
         odds, pvalue = stats.fisher_exact([[1, 2], [9, 84419233]])
 
 
+class TestBarnardExact(object):
```

Comment:
No need to change but inheriting from `(object)` is unnecessary as it's the default in Python 3.